File: 99_bottles.js
function bottleSong(bottles) {	
  let tmpBottles = Number(bottles);	
  while (tmpBottles > 2){	
    console.log(`${tmpBottles} bottles of beer on the wall, ${tmpBottles}	bottles of beer.\nTake one down and pass it around, ${tmpBottles - 1} bottles of beer on the wall.`);	
    tmpBottles --;	
  }	
  if (tmpBottles === 2){	
    console.log(`${tmpBottles} bottles of beer on the wall, ${tmpBottles}	bottles of beer.\nTake one down and pass it around, ${tmpBottles - 1} bottle of beer on the wall.`);	
    tmpBottles --;	
  }	
  if (tmpBottles === 1){	
  console.log(`1 bottle of beer on the wall, 1 bottle of beer.\nTake one down	and pass it around, no more bottles of beer on the wall.\nGo to the store and	buy some more, ${bottles} bottles of beer on the wall.`);	
  }	
};

bottleSong(99);

File: 99_bottles.py
def bottle_song(bottles):
	# write your code here!
    tmpBottles = bottles
    while tmpBottles > 2:
      print('%s bottles of beer on the wall. %s bottles of beer. Take one down, pass it around, %s bottles of beer on the wall.' % (tmpBottles, tmpBottles, tmpBottles-1))
      tmpBottles -=1
    print('%s bottles of beer on the wall. %s bottles of beer. Take one down, pass it around, %s bottle of beer on the wall.' % (tmpBottles, tmpBottles, tmpBottles-1))
    tmpBottles -=1
    print('1 bottle of beer on the wall. 1 bottle of beer. Take one down, pass it around, no more bottles of beer on the wall')
    print('No more bottles of beer on the wall, no more bottles of beer. Go to the store and buy some more, %s bottles of beer on the wall.' % bottles)

bottle_song(99)
